hyperparams/ppo2.yml

```diff
@@ -263,3 +263,33 @@ InvertedPendulumSwingupBulletEnv-v0:
   ent_coef: 0.0
   learning_rate: 2.5e-4
   cliprange: 0.2
+
+# following https://github.com/lcswillems/rl-starter-files
+# https://stable-baselines.readthedocs.io/en/master/modules/ppo2.html
+MiniGrid-DoorKey-5x5-v0:
+  normalize: true
+  n_envs: 8 # number of environment copies running in parallel
+  n_timesteps: !!float 1e5 # 80000 frames ?
+  policy: MlpPolicy # 'ActorCliticPolicy' or Mlp/Cnn/CnnLstmPolicy otherwise
```

Comment:
I think you need to add a custom policy to handle the observation properly
See the architecture in: https://github.com/lcswillems/rl-starter-files

Also related: https://github.com/hill-a/stable-baselines/issues/321

Reply:
Encoding 2d observation image (even small one) + optional mission text into a vector might be too rough.  Would like to do it sooner or later.

Reply:
I have just read through the Stable Baselines conversation.
So, we should take the vectorize-it-and-reshape-it approach.  Got it.

Reply:
Though custom policies are of my interest (to have the similar architecture to rl-starter-files), it is not quick to complete today.  Let me do this next week, hopefully as another PR.  :->
Your insight has been really helpful.  Thanks!

Reply:
your welcome ;)